analyzer/analyzer_test.go

```diff
@@ -14,31 +15,29 @@ import (
 func TestAnalyzer(t *testing.T) {
 	t.Parallel()
 	a := analyzer.New()
-	err := a.Flags.Set("err-error", "true")
-	if err != nil {
-		t.Fatalf("failed to set err-error flag")
-	}
-	analysistest.RunWithSuggestedFixes(t, analysistest.TestData(), a, "p")
-}
-
-func TestAnalyzerNoErrError(t *testing.T) {
-	t.Parallel()
-	a := analyzer.New()
-	err := a.Flags.Set("err-error", "false")
-	if err != nil {
-		t.Fatalf("failed to set err-error flag")
-	}
-	analysistest.RunWithSuggestedFixes(t, analysistest.TestData(), a, "nilerror")
-}
+	analysistest.RunWithSuggestedFixes(t, analysistest.TestData(), a, "default")
+	a.Flags.VisitAll(func(f *flag.Flag) {
+		if f.Name != "fiximports" {
```

Comment:
Please invert the logic


```suggestion
		if f.Name == "fiximports" {
			return
```

Code will be clearer and cleaner then

Reply:
Ok, looks the same to me